circle.yml

```diff
@@ -1,4 +1,9 @@
+# Note that the following stanza uses CircleCI 2.1 to make use of a Reusable Executor
+# This allows defining a docker image to reuse across jobs.
+# visit https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-executors to learn more.
```

Comment:
Can we remove this? Is it needed. Also should change the version below to `2.1` to be correct.

Reply:
removed